ReactViewControl/ViewModuleContainer.cs

```diff
@@ -58,7 +63,7 @@ public abstract class ViewModuleContainer : IViewModule {
 
         KeyValuePair<string, object>[] IViewModule.PropertiesValues => PropertiesValues;
 
-        void IViewModule.Bind(IFrame frame, IChildViewHost childViewHost) {
+        void IViewModule.Bind(IFrame frame, bool isHotReloadEnabled, IChildViewHost childViewHost) {
```

Comment:
instead of receiving isHotReloadEnabled by param add the property to the IChildViewHost

Reply:
If i'm correct and If I do this change, it won't work for plugins...

Reply:
Yes but reloading plugins is a risky thing...

Reply:
Ok, but it seems a little strange to load some files from the dll and others from webpack dev server. Your call, if you feel this is risky business I'll change it.

Reply:
I know ... but the need to change plugins are very low and in the majority of the cases it could have weird behaviors because plugins are meant to be loaded once only and cannot be "unloaded" safely